gnomad_qc/v4/resources/basics.py

```diff
@@ -157,14 +182,21 @@ def _ukbb_root_path() -> str:
 
 
 # List of samples to exclude from QC due to withdrawn consents
-# This is the file with the most up to date sample withdrawals we were sent. File downloaded on 8/16/21
-ukbb_excluded_samples_path = f"{_ukbb_root_path()}/w26041_20210809.csv"
+ukbb_excluded_26041_path = f"{_ukbb_root_path()}/w26041_20210809.csv"
```

Comment:
I know it's in the name, but add the date that this is from to the note like I mention above